src/mame/trs/coco12.cpp

```diff
@@ -644,30 +720,36 @@ ROM_END
 
 ROM_START(cocoe)
 	ROM_REGION(0x8000,MAINCPU_TAG,0)
-	ROM_LOAD("bas11.rom",   0x2000, 0x2000, CRC(6270955a) SHA1(cecb7c24ff1e0ab5836e4a7a8eb1b8e01f1fded3))
-	ROM_LOAD("extbas10.rom",    0x0000, 0x2000, CRC(6111a086) SHA1(8aa58f2eb3e8bcfd5470e3e35e2b359e9a72848e))
+	ROM_LOAD("bas11.rom",    0x2000, 0x2000, CRC(6270955a) SHA1(cecb7c24ff1e0ab5836e4a7a8eb1b8e01f1fded3))
+	ROM_LOAD("extbas10.rom", 0x0000, 0x2000, CRC(6111a086) SHA1(8aa58f2eb3e8bcfd5470e3e35e2b359e9a72848e))
 ROM_END
 
 ROM_START(coco2)
 	ROM_REGION(0x8000,MAINCPU_TAG,0)
-	ROM_LOAD("bas12.rom",   0x2000, 0x2000, CRC(54368805) SHA1(0f14dc46c647510eb0b7bd3f53e33da07907d04f))
-	ROM_LOAD("extbas11.rom",    0x0000, 0x2000, CRC(a82a6254) SHA1(ad927fb4f30746d820cb8b860ebb585e7f095dea))
+	ROM_LOAD("bas12.rom",    0x2000, 0x2000, CRC(54368805) SHA1(0f14dc46c647510eb0b7bd3f53e33da07907d04f))
+	ROM_LOAD("extbas11.rom", 0x0000, 0x2000, CRC(a82a6254) SHA1(ad927fb4f30746d820cb8b860ebb585e7f095dea))
+ROM_END
+
+ROM_START(deluxecoco)
+	ROM_REGION(0x8000,MAINCPU_TAG,0)
+	ROM_LOAD("bas20.rom",    0x2000, 0x2000, NO_DUMP)
+	ROM_LOAD("extbas20.rom", 0x0000, 0x2000, NO_DUMP)
 ROM_END
```

Comment:
Are the ROMs from the `coco2b` useful for testing this machine, even while the correct ROMs for it are undumped?  If they are, you might want to load them over the top of the `NO_DUMP` ROMs, just so people have something to play with.

Reply:
Implemented.